server/app/rest_api/rest.py

```diff
@@ -202,11 +205,108 @@ def get(self):
             return make_response(jsonify(annotation_response))
 
 
+class DataObsAPI(Resource):
+    @swagger.doc({
+        "summary": "Get data (expression values) from the dataframe.",
+        "tags": ["data"],
+        "parameters": [
+            {
+                "in": "query",
+                "name": "filter",
+                "type": "string",
+                "description": "axis:key:value"
+            },
+            {
+                "in": "query",
+                "name": "accept-type",
+                "type": "string",
+                "description": "MIME type"
+            },
+        ],
+        "responses": {
+            "200": {
+                "description": "expression",
+                "examples": {
+                    "application/json": {
+                        "var": [0, 20000],
+                        "obs": [
+                            [1, 39483, 3902, 203, 0, 0, 28]
+                        ]
+                    }
+                }
+            },
+            "400": {
+                "description": "Malformed filter"
+            },
+            "406": {
+                "description": "Unacceptable MIME type"
+            },
+        }
+    })
+    def get(self):
+        # request.args is immutable
+        args = dict(request.args)
+        accept_type = args.pop("accept-type", None)
```

Comment:
why are we accepting this as an arg and not as a header?

Reply:
So that we can have deep linking URLs. We have changed the spec so that we will take accept-type query param first, if missing then header, and then finally JSON if both are missing. But, that level isn't implemented yet.

Reply:
this is kind of a odd design.  I would prefer just a format arg that explicitly says json or whatever type, and that overrides whatever the accept-type header presents.

Reply:
Let's talk in person. I'm not sure I know what you are advocating for.